crates/core_simd/src/vector.rs

```diff
@@ -12,7 +12,39 @@ pub(crate) mod ptr;
 use crate::simd::intrinsics;
 use crate::simd::{LaneCount, Mask, MaskElement, SupportedLaneCount};
 
-/// A SIMD vector of `LANES` elements of type `T`.
+/// A SIMD vector of `LANES` elements of type `T`. `Simd<T, N>` has the same shape as [`[T; N]`](array), but operates like `T`.
+/// This type is commonly known by names like `f32x4` or `Vec4` in many programming languages.
```

Comment:
Not sure I would bother including this line, there are so many different languages with so many different names

Reply:
Hm. I was originally thinking "well, we use the names for our aliases..." but even that's not set in stone, and would be better pointed out in those terms if we did decide so, so I think you're right to be uncertain.